staging/src/k8s.io/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -142,10 +144,13 @@ func (o AggregatorOptions) RunAggregator(stopCh <-chan struct{}) error {
 	if err != nil {
 		return err
 	}
+	kubeInformers := kubeinformers.NewSharedInformerFactory(coreAPIServerClient, 5*time.Minute)
+	serviceResolver := apiserver.NewClusterIPServiceResolver(kubeInformers.Core().V1().Services().Lister())
```

Comment:
> This should respect the flag, IMO.

There isn't a flag when run as a separate processes since its probably in a pod which would use `kube-proxy`.